paddle/fluid/operators/ngraph/ngraph_engine.cc

```diff
@@ -0,0 +1,495 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <glog/logging.h>
```

Comment:
> The code is located under operators/ngraph. The NgraphEngine op is mostly moved from framework/ngraph_operactor.. After this PR is approved. ngraph_operator. can be deleted and ngraph_bridge.* can be moved to oeprator/ngraph as well.

Why not delete ngraph_operator and move ngraph_bridge.* to `operator/ngraph` at the same time?